src/gfx/shaders/Uber_vert.glsl

```diff
@@ -218,12 +219,19 @@ void main() {
 
 #if defined(USE_LIGHTS) && defined(SHADOWMAP)
 	#if NUM_DIR_LIGHTS > 0
-	vec4 worldPosition;
-	#pragma unroll_loop
-	for ( int i = 0; i < NUM_DIR_LIGHTS; i ++ ) {
-	  worldPosition = vec4(vWorldPosition, 1.0);
-		vDirectionalShadowCoord[ i ] = directionalShadowMatrix[ i ] * worldPosition;
-	}
+	  vec4 worldPosition;
+	  #pragma unroll_loop
+	  for ( int i = 0; i < NUM_DIR_LIGHTS; i ++ ) {
+	    #ifdef NORMAL_OFFSET_BIAS
+        vec4 worldNormal = modelMatrix * vec4(objectNormal, 0.0);
+        float bias = 0.09; // smaller bias -> more self-shadowing
```

Comment:
Is bias a constant value?  But what about bias from directionalLight[i].shadowBias?

Reply:
It was extra code. I removed it